### DISCUSSION TITLE
Various pop-up menus cannot be retracted and may display incorrectly, in some browsers.

### DISCUSSION THREAD

**Discussion by JunmoreHeavyBox** [community/community#78114] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Various pop-up menus cannot be closed and retracted and may display incorrectly, on the Vivo's browser.
![Screenshot_20231207_150021](https://github.com/community/community/assets/114210303/532d322f-8f43-4cda-a79e-d0c063119c8f)

**Comment by Khushiyant** [comment]
The issue that you are facing with pop-up menus on the Vivo's browser is a known one. There are a few things that you can try to fix it:

- **Update your browser.** Make sure that you are using the latest version of the Vivo's browser. The developers may have released a fix for this issue in a recent update.
- **Clear your browser's cache and cookies**. Sometimes, corrupted cache and cookies can cause problems with websites and apps. To clear your browser's cache and cookies, open the Vivo's browser and go to Settings > Privacy and security > Clear browsing data.
- **Check your browser's permissions.** Make sure that the Vivo's browser has permission to open pop-up menus. To check your browser's permissions, open the Vivo's browser and go to Settings > Sites > Permissions. Under Pop-ups and redirects, make sure that the Allow all pop-ups and redirects toggle is switched on.

Here are some additional tips for troubleshooting pop-up menus:

- Make sure that you are not running any ad blockers or pop-up blockers. These extensions can sometimes interfere with the operation of pop-up menus.
- Try disabling any extensions that you are not using. Sometimes, extensions can conflict with each other and cause problems.

**Comment by JunmoreHeavyBox** [comment]
Browser is up to date 17.5.0.0, cache and cookies cleared, and tried turning off adblock. But the problem is still there.

Additional Info:
FuntouchOS9 v1.23.17

The vivo browser here refers to the browser that comes with the phone system.

**Comment by JunmoreHeavyBox** [comment]
I think the problem has been gradual resolved in the recent update.
